CriticalMass/MapViewController.swift

```diff
@@ -41,8 +41,8 @@ class MapViewController: UIViewController {
 
     private var mapView = MKMapView(frame: .zero)
 
-    private let gpsDisabledOverlayView: BlurryOverlayView = {
-        let view = BlurryOverlayView.fromNib()
+    private let gpsDisabledOverlayView: BlurryFullscreenOverlayView = {
```

Comment:
This would be a perfect candiate for the PropertyWrapper that we had earilier :)

Reply:
What was the property wrapper again? I remember that there was a cool one but can't find it anymore

Reply:
It’s gone. Made us loose the translates... (unsure how the property is called) property. Maybe I’ll add it again and adopt it to other properties too

Reply:
... in another PR